### DISCUSSION TITLE
How do use Nuxt Picture with different formats?

### DISCUSSION THREAD

**Discussion by jamesxu123** [nuxt/image#672] [Q&A]
I'd like to use the Nuxt Picture tag to be able to dynamically convert images to several formats, e.g. have AVIF, WebP, and JPEG for fallback. I couldn't find an example for this. How should I do this?

**Comment by marcelxpfeifer** [comment]
You currently can't there is an issue open for that: [451](https://github.com/nuxt/image/issues/451)
